ipn/ipnlocal/local.go

```diff
@@ -3636,6 +3636,12 @@ func (b *LocalBackend) authReconfig() {
 	}
 	b.logf("[v1] authReconfig: ra=%v dns=%v 0x%02x: %v", prefs.RouteAll(), prefs.CorpDNS(), flags, err)
 
+	if b.netMap.SelfNode.HasCap(tailcfg.NodeAttrUserDialUseRoutes) {
```

Comment:
i think you want to use `b.netmap.HasCap`

Reply:
Also `b.netMap` is guarded by `b.mu` which isn't held right now, so this is a race.

But `nm` holds `b.netMap`, so use `nm.HasCap`

Reply:
Done, thanks!